.golangci.yml

```diff
@@ -1,18 +1,21 @@
 # Documentation: https://golangci-lint.run/usage/configuration/
 linters-settings:
   errcheck:
-    exclude: .errcheck.txt
+    exclude-functions:
+      - (*github.com/tektoncd/pipeline/vendor/go.uber.org/zap.SugaredLogger).Sync
+      - flag.Set
+      - os.Setenv
+      - logger.Sync
+      - fmt.Fprintf
+      - fmt.Fprintln
+      - (io.Closer).Close
+      - updateConfigMap
   gomodguard:
     blocked:
       modules:
         - github.com/ghodss/yaml:
             recommendations:
               - sigs.k8s.io/yaml
-  depguard:
-    list-type: denylist
-    include-go-root: false
-    packages-with-error-message:
-    - k8s.io/apimachinery/pkg/util/clock: 'use k8s.io/utils/clock or k8s.io/utils/clock/testing'
```

Comment:
thanks for cleaning this up, this was introduced in PR #5498 to make sure we avoid using apimachinery clock. I am assuming its now safe to drop this since we have transitioned to the desired clock now - `k8s.io/utils/clock`

Reply:
👍